doc_source/virtual_gateways.md

```diff
@@ -2,7 +2,7 @@
 
 A virtual gateway allows resources that are outside of your mesh to communicate to resources that are inside of your mesh\. The virtual gateway represents an Envoy proxy running in an Amazon ECS service, in a Kubernetes service, or on an Amazon EC2 instance\. Unlike a virtual node, which represents Envoy running with an application, a virtual gateway represents Envoy deployed by itself\. 
 
-External resources must be able to resolve a DNS name to an IP address assigned to the service or instance that runs Envoy\. Envoy can then access all of the App Mesh configuration for resources that are inside of the mesh\. To complete an end\-to\-end walkthrough, see [Configuring Ingress Gateway](https://github.com/aws/aws-app-mesh-examples/tree/master/walkthroughs/howto-ingress-gateway)\.
+External resources must be able to resolve a DNS name to an IP address assigned to the service or instance that runs Envoy\. Envoy can then access all of the App Mesh configuration for resources that are inside of the mesh\. The configuration for handling the incoming requests at the Virtual Gateway are specified using [Gateway Routes](https://docs.aws.amazon.com/app-mesh/latest/userguide/gateway-routes.html). By default, Virtual Gateways with HTTP and HTTP2 listeners rewrite the incoming request's hostname to the Gateway Route target Virtual Service's name and the matched prefix from the Gateway Route is rewritten to `/` \. To complete an end\-to\-end walkthrough, see [Configuring Ingress Gateway](https://github.com/aws/aws-app-mesh-examples/tree/master/walkthroughs/howto-ingress-gateway)\.
```

Comment:
Would it make sense to provide an example rewrite for clarity?

I think it might also sense to call this out in a separate paragraph to make it more visible.

Reply:
We have an example already in the [Gateway route page](https://docs.aws.amazon.com/app-mesh/latest/userguide/gateway-routes.html) under "To create a gateway route". Do you feel that we might need to pull it up here?

Reply:
If we're mentioning it at all here, I think we should provide an example -- the sentence by itself is still up to some interpretation by the reader.

Reply:
Added a minimal example. Have also added a reference to the section in Gateway route which has multiple examples for the same.